cmd/cloud/cluster.go

```diff
@@ -7,6 +7,7 @@ import (
 )
 
 func init() {
+	clusterCreateCmd.Flags().String("cluster", "", "The id of the cluster to be created. A random id will be used if left empty.")
```

Comment:
Contrary to my own scratch notes on the associated ticket, I'm not entirely convinced we should let this be specified by the user. The only practical use case I can imagine is to support idempotency. But even if we do want to support that, I'd propose not doing it at the primary key level (e.g. leave the application to pick those values), and instead allow some other unique column like a "friendly cluster name" (e.g. `nod-bygone-hyped`). Either way, maybe a future feature?

Reply:
I would like to keep this for now. My thinking is it could speed up testing if we are able to create clusters with friendly IDs or even to just speed up scripting by creating clusters with the same ID every time.

Also, perhaps we would want to leave this as a command line option, but not as an API call later. That way it could be used by developers as a tool in a more powerful way.

Thoughts?

Reply:
We chatted about this and agree that we will move forward with no custom IDs for now. If anything, we will add naming metadata in the future.